contributed_definitions/nyaml/NXheater.yaml

```diff
@@ -0,0 +1,22 @@
+category: base
+doc: |
+  A device for heating.
+type: group
+NXheater(NXobject):
```

Comment:
Should eventually inherit from NXactuator or at least have a similar structure.

Reply:
To some extend I am supportive of your argumentation.
If you have a furnace say with an resistive wire severing as a heater it is a substantiated argument to say that the entire furnace is not necessarily the resistive wire (neither at or nor alone) and thus there can be a base class furnace which has, I fully support an instance of an NXactuator. The heater meant here is a heating chip used in e.g. a TEM that is more than a resistive wire (if it is a wire at all) is it an entire furnace essentially. I have complained since two years that there has not been a base class built for describing something as simple as a a furnace we should change that but in NXheater or NXfurnace use indeed NXactuator.

While walking through NXactuator I found some points thinking about eventually:
"used to control an external condition" a resistive wire does not control anything, there is just a current pumped through it or not, hence an actuator with is modelled as that it can control s.th. has logics on it in addition to functional actuating units. I like it the way it is like this but one could also state that the logic is elsewhere and actuator is just a part.

Reply:
Ok, let's leave it as it is right now.

Agree that some dostrings in NXactuator could be improved if we are widening the scope of what we mean by an actuator. Feel free to make a suggestions for changing NXactuator when you use it somewhere where this may come up.